### PR TITLE
Instructions updated in md files

### PR DESCRIPTION
**Pull request by PuneetKMinhas**
# Summary
Files Changed
- ReadMe.MD
- Lab4.md

# To Do:

- [ ] Include links to workshop pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +58,5 @@ After you have completed this course, you are probably wondering where to go fro
 - [GitHub Learning Lab](https://lab.github.com/): self-paced courses with instant bot-assisted feedback.
 - [Microsoft Learn for GitHub](https://docs.microsoft.com/en-us/learn/github/)
 - [Join the open source community](https://github.com/open-source)
+
+If you are interested in further workshops, please speak to your MS Account Team.
```
Comment (Puneet-Minhas) [inline README.md:62]: this is good. 😄

File: docs/day4.md [hunk 2]
```diff
@@ -13,3 +13,4 @@
 
 ### After Class Activities:
 - None! But, remember you can always ask more questions in 1:1 sessions or through issues. :sparkles:
+- There are similar workshops available for GitHub Actions and GitHub Copilot!
```
Comment (Puneet-Minhas) [inline docs/day4.md:16]:
```suggestion
- There are similar workshops available for GitHub Actions and GitHub Copilot!
Please speak to your account team for further information.
```

**Review by Puneet-Minhas — CHANGES_REQUESTED** [review]
Please go through my notes. Let me know if you need any further info.

**Review by PuneetKMinhas — COMMENTED** [review]
suggestion incorporated. please review

File: docs/day4.md [hunk 3]
```diff
@@ -13,3 +13,5 @@
...
+Please speak to your account team for further information.
```
Comment (Puneet-Minhas) [inline docs/day4.md:17]:
```suggestion
Please speak to your account team for further information.
Contact: <todo>
```

**Review by Puneet-Minhas — CHANGES_REQUESTED** [review]

File: docs/day4.md (hunk 3 above)
Comment (PuneetKMinhas) [reply docs/day4.md:17]: Incorporated.

**Review by Puneet-Minhas — APPROVED** [review]

**Comment by Puneet-Minhas** [comment]
not going ahead

**Pull request closed without merging**